.github/dependabot.yml

```
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    open-pull-requests-limit: 6
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "bot: dependencies update"
+    reviewers:
+      - "wordpress-mobile/android-developers"
+    ignore:
+      # The Android Gradle Plugin is a dependency that needs to be in sync with other
+      # in-house libraries due to compatibility with composite build.
+      - dependency-name: "com.android.tools.build:gradle"
+      # Bumping 1.2.1 to 1.3.0 causes some issues, fist spotted in Reader. For more details, see
+      # https://github.com/wordpress-mobile/WordPress-Android/pull/14431
+      - dependency-name: "com.google.android.material:material"
```

Something like this, except be sure to update the link to the issue once you've created it.
```suggestion
      # Bumping 1.2.1 to 1.3.0 causes some issues, first spotted in Reader. For more details, see
      # https://github.com/wordpress-mobile/WordPress-Android/pull/14431
      # So we plan to handle the update manually later, to handle it in correlation with `Constraint Layout`; see
      # https://github.com/wordpress-mobile/WordPress-Android/issues/<link-to-issue>
      - dependency-name: "com.google.android.material:material"
```